views/partials/header.hbs

```diff
@@ -7,7 +7,7 @@
     </div>
   {{/if}}
   <div class="chooseLang">
-    <label><input id="enLang" type="radio" name="optradio">English</label>
-    <label><input id="arLang" type="radio" name="optradio">Arabic</label>
+    <label><input id="enLang" type="checkbox" name="optradio">English</label>
```

Comment:
What was the reason for changing these from radio buttons? I think the user should only be allowed to select one at once.

Reply:
we didn't want inputs now , I changed to two buttons Arabic, English in another branch

Reply:
@JWLD ohh sorry about that i was testing something and forgot to remove them